programs/svm-spoke/src/instructions/fill.rs

```diff
@@ -179,27 +161,18 @@ pub struct CloseFillPda<'info> {
         bump,
         close = signer, // TODO: check if this is correct party to receive refund.
         // Make sure caller provided relay_hash used in PDA seeds is valid.
-        constraint = is_relay_hash_valid(&relay_hash, &relay_data, &state) @ CustomError::InvalidRelayHash
+        constraint = is_relay_hash_valid(&relay_hash, &relay_data, &state) @ SvmError::InvalidRelayHash
     )]
     pub fill_status: Account<'info, FillStatusAccount>,
 }
 
-pub fn close_fill_pda(
-    ctx: Context<CloseFillPda>,
-    _relay_hash: [u8; 32], // TODO: check if we can use underscore in functions while in context macro leave as is?
-    relay_data: V3RelayData
-) -> Result<()> {
+pub fn close_fill_pda(ctx: Context<CloseFillPda>, _: [u8; 32], relay_data: V3RelayData) -> Result<()> {
```

Comment:
we can delete the unused prop here and rename to `_relay_hash` in the outer function.